daemon/inspect.go

```diff
@@ -151,3 +166,30 @@ func (daemon *Daemon) VolumeInspect(name string) (*types.Volume, error) {
 	}
 	return volumeToAPIType(v), nil
 }
+
+func getBackwardsCompatibleNetworkSettings(settings *network.Settings) *v1p20.NetworkSettings {
```

Comment:
So if you used the inspect command you described, it will fail because there is no such IPAddress field on API 1.21.